packages/material-top-tabs/src/utils/useTabAnimation.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+import { TabAnimationContext } from './TabAnimationContext';
+
+export function useTabAnimation() {
+  const animation = React.useContext(TabAnimationContext);
+
+  if (animation === undefined) {
+    throw new Error(
+      "Couldn't find values for card animation. Are you inside a screen in Tab?"
```

Comment:
i believe it's worth clarifying that it's material-top-tabs (and not bottom-tabs).

Reply:
Also, `card animation`? 😅